inc/db.php

```diff
@@ -9,27 +10,33 @@ public static function init() {
 			Config::$dbPassword
 		);
 		static::$conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+		static::$isMysql = static::$conn->getAttribute(PDO::ATTR_DRIVER_NAME) == 'mysql';
 		static::createTables();
 	}
 
 	private static function createTables() {
 		static::$conn->exec('
 			CREATE TABLE IF NOT EXISTS packages (
-				PackageId TEXT PRIMARY KEY,
-				Title TEXT,
+				PackageId ' . (static::$isMysql ? 'VARCHAR (256)' : 'TEXT PRIMARY KEY') . ',
+				Title ' . (static::$isMysql ? 'VARCHAR (256)' : 'TEXT') . ',
 				DownloadCount INTEGER NOT NULL DEFAULT 0,
-				LatestVersion TEXT
+				LatestVersion TEXT' . (static::$isMysql ? ',
+				PRIMARY KEY (PackageId),
```

Comment:
I think you can inline this in the column definition (eg. `PackageId VARCHAR(256) PRIMARY KEY`)